Makefile

```diff
@@ -237,7 +237,7 @@ test-sim:
 	@VERSION=$(VERSION) go test -mod=readonly $(PACKAGES_SIM)
 
 test-e2e:
-	@VERSION=$(VERSION) go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E)
```

Comment:
Wdyt about only having `OSMOSIS_E2E_FORK_HEIGHT`?

If it is set, then we can set `s.isFork` to true in `e2e_setup_test.go`. If not, we can keep it as default false.

The boolean seems redundant because having a height is idiomatic with the True value of `OSMOSIS_E2E_IS_FORK`